CommandLine.Tests/HelpViewTests.cs

```diff
@@ -303,5 +303,36 @@ public void When_unmatched_tokens_are_allowed_then_help_view_indicates_it()
 
             helpView.Should().Contain("Usage: some-command [options] [[--] <additional arguments>...]]");
         }
+
+        [Fact]
+        public void Retain_single_dash_on_multi_char_option_when_display_raw_aliases_is_specified()
+        {
+            var command = Command("command", "Help Test",
+                    Option("-multi|--alt-option", "Help for option"));
+            HelpViewOptions helpViewOptions = new HelpViewOptions(true);
+            var helpView = command.HelpView(helpViewOptions);
+            helpView.Should().Contain("-multi");
+            helpView.Should().NotContain("--multi");
+        }
```

Comment:
I'm not sure this actually requires introducing a `HelpViewOptions` abstraction because showing `--multi` looks like it would just be a bug for this configuration.